lib/schema/schema.graphql

```diff
@@ -359,6 +360,7 @@ type User {
   relationships: [UserRelationship!]
   inactivatedAt: DateTimeISO
   inactivatedBy: ID
+  termsAndConditionsAcceptedAt: DateTimeISO
```

Comment:
Just an FYI - didn't realize this was already merged in the backend. I envisioned using this on [the signup/onboarding screen](https://github.com/micromentor-team/mm-flutter-app/blob/29350a623750e54bac3474f762c90b00960d859d/lib/widgets/features/sign_up/sign_up_method.dart) where the user accepts some conditions about MM.

Not sure if that's relevant given that we only run the `updateUser` graphql at the end of the signup, rather than on each screen.